Flashcards/task/src/flashcards/Main.java

```diff
@@ -47,133 +90,248 @@ private void runGameByAction() {
             case "exit":
                 exitGame();
                 break;
+            case "log":
+                exportLog();
+                break;
+            case "hardest card":
+                hardestCard();
+                break;
+            case "reset stats":
+                resetStats();
+                break;
             default:
                 break;
         }
     }
 
+    private void fillCardMap() {
```

Comment:
methode naam is nu gelinkt aan je type waarin je de kaarten opslaat. Probeer dit te vermijden. Je hebt dit nu namelijk aangepast en het is geen map meer, dus het staat raar.

Reply:
Goeie, done.